### ISSUE TITLE
Snippets generation failure for Powershell

### ISSUE THREAD

**Issue by adhiambovivian** [microsoftgraph/microsoft-graph-explorer-v4#1739]
Bug: Code snippets are not generating for https://graph.microsoft.com/v1.0/organization/XXXX/branding/localizations for Powershell

![image](https://user-images.githubusercontent.com/9458079/168818454-6476bab1-35a9-4983-91e4-26fc0410e4ea.png)

**Comment by millicentachieng** [comment]
The PowerShell command for the query, which would be `Get-MgOrganizationBrandingLocalization -OrganizationId $organizationId`, is not available on PowerShell SDK. 

This is not a bug but a limitation of the PowerShell SDK.

**Comment by adhiambovivian** [comment]
thank you @millicentachieng  what about this for both go and powershell? This path was removed from openAPI desc doc, am not sure if this could be what's causing the unavailability of snippets for this path 

![image](https://user-images.githubusercontent.com/9458079/169233780-f15fa550-fed8-406c-8b9a-80dff001e196.png)

**Comment by RabebOthmani** [comment]
Adding more examples where the generation fails : 
GET /users/640885de-9652-4fb2-8a87-963cc2f599a0/photos/48x48/$value

GET https://graph.microsoft.com/beta/users/e436ca15-3a39-4dcc-819e-7dbb246cd46b/memberOf/microsoft.graph.administrativeUnit
GET administrativeUnits?$select=id,displayName,description&$top=51
GET /groups/6565dae0-a004-4464-aedb-a41569bfdb59/photos/48x48/$value
GET https://graph.microsoft.com/beta/users/e436ca15-3a39-4dcc-819e-7dbb246cd46b/memberOf/$/microsoft.graph.group?$select=id,displayName,mailEnabled,securityEnabled,groupTypes,onPremisesSyncEnabled,mail,isAssignableToRole&$top=20&$orderby=displayName%20asc&$count=true
GET /groups/0a15764d-b8b5-462a-b7f0-f93e229ce246/transitivemembers/$count

**Comment by millicentachieng** [comment]
@adhiambovivian, @RabebOthmani  PowerShell snippets for the following paths are now being generated
- https://graph.microsoft.com/v1.0/identityProtection/riskyUsers?$filter=riskLevel eq 'high'
- https://graph.microsoft.com/v1.0/identityProtection/riskDetections

These are not supported for PowerShell only
- https://graph.microsoft.com/v1.0/me/transitiveMemberOf/microsoft.graph.group?$count=true
- https://graph.microsoft.com/v1.0/me/calendarview
- https://graph.microsoft.com/v1.0/me/calendarview/delta
- https://graph.microsoft.com/beta/me/insights/trending

These paths are not generated in the Open  API document and thus wont work for PowerShell and Go.
- https://graph.microsoft.com/v1.0/me/drive/items/{drive-item-id}/workbook/worksheets
- https://graph.microsoft.com/v1.0/me/drive/items/{drive-item-id}/workbook/worksheets('Sheet1')/usedRange
- https://graph.microsoft.com/v1.0/me/drive/items/{drive-item-id}/workbook/worksheets/Sheet1/tables
- https://graph.microsoft.com/v1.0/me/drive/items/{drive-item-id}/workbook/worksheets('Sheet1')/charts
- https://graph.microsoft.com/v1.0/groups/{group-id}/drive/items/root/children
- https://graph.microsoft.com/v1.0/me/drive/recent 
- https://graph.microsoft.com/v1.0/groups/{group-id-for-teams}/drive/items/root/children
- https://graph.microsoft.com/v1.0/me/drive/root/children
- https://graph.microsoft.com/v1.0/me/drive/sharedWithMe
- https://graph.microsoft.com/v1.0/me/drive/root/search(q='finance')?select=name,id,webUrl
- https://graph.microsoft.com/v1.0/groups/0a15764d-b8b5-462a-b7f0-f93e229ce246/transitivemembers/$count
- https://graph.microsoft.com/beta/users/e436ca15-3a39-4dcc-819e-7dbb246cd46b/memberOf/$/microsoft.graph.group
- https://graph.microsoft.com/v1.0/groups/6565dae0-a004-4464-aedb-a41569bfdb59/photos/48x48/$value
- https://graph.microsoft.com/v1.0/administrativeUnits?$select=id,displayName,description&$top=51
- https://graph.microsoft.com/v1.0/users/640885de-9652-4fb2-8a87-963cc2f599a0/photos/48x48/$value
- https://graph.microsoft.com/beta/users/e436ca15-3a39-4dcc-819e-7dbb246cd46b/memberOf/microsoft.graph.administrativeUnit

We do have a gap in the Open API path generation where not all paths are created during the conversion from CSDL.

**Comment by millicentachieng** [comment]
Meanwhile, we do have this [ticket](https://github.com/microsoftgraph/microsoft-graph-devx-api/issues/1047) on DevX API that will enable us to specify which paths, entities or singletons to skip for PowerShell only without affecting other Open API generated code snippets.

**Comment by ddyett** [comment]
@timayabi2020 can you have look at where we drop the navigationproperty for beta instances of users/id/insights/trending.

There are paths that should be expected to fail so we need to make sure we have the right experience around those expected failures. I don't think we should be skipping paths we should have the right experience or investigate the source of the failure like the one I've asked Tim to look at.

The other aspect of this is there's a gap between raptor validation and the total number of openapi paths.  This means we can't validate all of this, so we need to look at Raptor can be run based off openapi paths instead of available docs only.  @zengin

**Comment by adhiambovivian** [comment]
@ddyett  do we have a documented list of paths that powershell does not support by 'design' ? So far, I know that Powershell does not support /me paths and is this information available to users?

**Comment by timayabi2020** [comment]
- [ ] https://graph.microsoft.com/beta/me/insights/trending. Issue raised [here](https://github.com/microsoft/OpenAPI.NET.OData/issues/268)

**Comment by RabebOthmani** [comment]
@millicentachieng I know you've been doing work on this on the DevX API side, what's the status ? 
@ddyett Do we want to change the label or anything on this issue? It has been around since August as a hotbug

**Comment by darrelmiller** [comment]
@timayabi2020 Could you create a new issue in the appropriate repo so that this issue can be closed as it is not something to be fixed in Graph Explorer?  Thanks.

**Comment by RabebOthmani** [comment]
@adhiambovivian do we still need this issue open?

**Comment by adhiambovivian** [comment]
Yes @RabebOthmani , it should however be transferred to Powershell repo.
